lib/Installation/Popups/PopupController.pm

```diff
@@ -24,23 +24,23 @@ sub new {
 
 sub init {
     my ($self) = @_;
-    $self->{ConfirmationWarningRichText} = Installation::Warnings::ConfirmationWarningRichText->new({app => YuiRestClient::get_app()});
+    $self->{Popup} = Installation::Popups::OkPopup->new({app => YuiRestClient::get_app()});
     return $self;
 }
 
-sub get_confirmation_warning_rich_text {
+sub get_popup {
     my ($self) = @_;
-    return $self->{ConfirmationWarningRichText};
+    return $self->{Popup};
 }
 
 sub get_text {
     my ($self) = @_;
-    $self->get_confirmation_warning_rich_text->text();
+    $self->get_popup->text();
 }
 
 sub accept_warning {
```

Comment:
I guess, we can even use `accept`, as "popup" word already contained in `get_popup`. WDYT?

Reply:
Yes those suggestions make more sense now. If it's ok I will address this in my followup PR, which is about rewriting the `PopupController` and changing some of it's functions, including this one.

Reply:
`accept` should be a good name, you can still leave it consistent in this PR, that is just about renaming, the other one you prepare is more complex, it might take more time to merge or could potentially be reverted in the worse case (sometimes happens). wdyt?

Reply:
Ok!